Tasks/Common/RemoteDeployer/Tests/Retry-Connection.AttemptsConnectionStateDisconnectedAvailabilityNone.ps1

```diff
@@ -0,0 +1,30 @@
+[CmdletBinding()]
+param()
+
+# Arrange.
+. $PSScriptRoot\..\..\..\..\Tests\lib\Initialize-Test.ps1
+$module = Microsoft.PowerShell.Core\Import-Module $PSScriptRoot\.. -PassThru
+
+$tms = @(
+    @{
```

Comment:
Can you please put this object creation in Test-common by just passing computerName ?